internal/v1/handler_post_compose_test.go

```diff
@@ -2961,6 +2954,12 @@ func TestComposeCustomizations(t *testing.T) {
 							Type:    v1.UploadTypesAwsS3,
 							Options: uo,
 						},
+						AAPRegistration: &v1.AAPRegistration{
```

Comment:
@lzap is this what you had in mind?

I can squash these commits into the previous ones if this is the route we want to go

Reply:
Yes, tho, we need to hear from others.